### ISSUE TITLE
Volume Control for Mic Feeds

### ISSUE THREAD

**Issue by bigbluebutton-issue-import** [bigbluebutton/bigbluebutton#1043]
Originally reported on Google Code with ID 288

```
By default, BBB does not give users the ability to adjust mic levels, of
the other users feeds.

Would be great if there was a slider control added.
```

Reported by `Tilton.John` on 2009-12-30 07:20:40

**Comment by MarcusWolschon** [comment]
> 
> 
> ```
> The idea was that you shouldn't need yet another volume control, since you can always
> adjust the volume on your system controls.
> ```
> 
> Reported by `Me.Snap` on 2009-12-30 14:14:59

Yes, you need one. Because it's not the only program on the computer playing audio.

**Comment by flyingKaenguru** [comment]
I absolutely agree - even the people who actually know about technology are sometimes too quiet or too loud in my company... It's just annoying now. If someone knows a programming solution please let me know.

**Comment by MarcusWolschon** [comment]
Why has this issue been merged into #122 "Resize HTML5 whiteboard "?
They have absolutely nothing in common.

**Comment by hex-m** [comment]
> ```
> Merging this into #122 -- pre-flight checklist
> 
> BigBlueButton should give the user the ability to check their voice before joining
> the voice conference.  This way, they can hear if their audio is too low (from an echo
> test) and increase their gain.
> ```

There is something wrong with the issue importer. According to the mentioned name this should be merged into #877.
